### PR TITLE
Fixed https://github.com/google/personfinder/issues/85

### PR DESCRIPTION
**Pull request by ezbr**
Fixed this issue: https://github.com/google/personfinder/issues/85

### CODE REVIEW

File: app/flag_note.py [hunk 1]
```diff
@@ -57,6 +57,7 @@ def post(self):
             note.source_date = now
             note.entry_date = now
             db.put(note)
+            note.put()
```
Comment (gimite) [inline app/flag_note.py]: As chatted offline, I believe this doesn't fix the issue completely.

Comment (ezbr) [inline app/flag_note.py]: Done.

**Comment by gimite** [comment]
@ezbr Is this ready for review? Did you verify that it works?

**Comment by ezbr** [comment]
I verified that this fixes the issue. Please review it.
Best,
Evan

On Mon, Aug 31, 2015 at 12:40 PM, Hiroshi Ichikawa <notifications@github.com

**Comment by ezbr** [comment]
Oops, I thought I was requesting to merge, but I think it just merged. I'm
sorry. Should I revert it?

**Comment by gimite** [comment]
LGTM

**Pull request merged**